Makefile

```diff
@@ -190,6 +190,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/oauth && $(KUSTOMIZE) edit set image oauth=${SPIS_IMG} && cd ../..
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+deploy_k8s: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${SPIO_IMG} && cd ../..
+	cd config/oauth && $(KUSTOMIZE) edit set image oauth=${SPIS_IMG} && cd ../..
+	$(KUSTOMIZE) build config/k8s | kubectl apply -f -
+
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
```

Comment:
Shouldn't there be a dedicated `undeploy_k8s`, too?

Reply:
done